gradle/libs.versions.toml

```diff
@@ -0,0 +1,31 @@
+[libraries]
+
+log4jApi = 'org.apache.logging.log4j:log4j-api:2.17.1'
+log4jCore = 'org.apache.logging.log4j:log4j-core:2.17.1'
+commonsLang = 'org.apache.commons:commons-lang3:3.12.0'
+guava = 'com.google.guava:guava:19.0'
```

Comment:
This should be 20.0. Perhaps that's a job for a different MR, but every single indeed deployable is currently using 20.0 or 31.1-jre, so building and testing this against 19 will allow some errors to slip through. I'd recommend committing 20.0, but testing 31.1-jre locally (without committing it) just to ensure it's able to compile and run unit tests using both.

Reply:
Bumped version to 20.0.

Tested locally with 31.1-jre and no issues seen.